### DISCUSSION TITLE
How to call a function using Liquid Expressions ?

### DISCUSSION THREAD

**Discussion by minhhoangyeu** [elsa-workflows/elsa-core#2440] [Q&A]
Hi @sfmskywalker,
I want to use the Liquid expression to call a function. Can you help me with how to do it?

Eg :

I have 1 function Calculation(int a,int b) to calculate.
I want to use the expression to call this calculation function {{ Calculation(3,4) }}

![How](https://user-images.githubusercontent.com/39401152/138226461-3082adeb-60d0-447d-941b-5b4092bff052.PNG)

Thank you!

**Comment by sfmskywalker** [comment]
Sure! In Liquid, the way to do that is to implement custom filters.

One example can be found here: https://elsa-workflows.github.io/elsa-core/docs/next/extensibility/extensibility-liquid#custom-liquid-filters

For more details, checkout Fluid's documentation here: https://github.com/sebastienros/fluid#custom-filters
